dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -47,6 +47,9 @@ RUN pip3 install requests
 # We install the libpci module in order to be able to do PCI transactions
 RUN pip3 install libpci
 
+# Install python-redis
+RUN pip3 install redis==4.5.4
```

Comment:
This is already in base image docker-config-engine-bullseye*

Reply:
Fixed, code removed.